P5/Source/Specs/citeStructure.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="header" ident="citeStructure">
+  <gloss versionDate="2020-02-03" xml:lang="en">citation structure</gloss>
+  <desc versionDate="2020-02-03" xml:lang="en">declares a structure and method for citing the current document.</desc>
+  <classes>
+    <memberOf key="att.global"/>
+    <memberOf key="att.citeStructurePart"/>
+  </classes>
+  <content>
+    <sequence>
+      <elementRef key="citeData" minOccurs="0" maxOccurs="unbounded"/>
+      <elementRef key="citeStructure" minOccurs="0" maxOccurs="unbounded"/>
+    </sequence>
+  </content>
+  <attList>
+    <attDef ident="delim" usage="opt">
+      <gloss versionDate="2020-02-03" xml:lang="en">delimiter</gloss>
+      <desc versionDate="2020-02-03" xml:lang="en">supplies a delimiting string preceding the structural component.</desc>
+      <datatype>
+        <dataRef name="string" restriction=".+"/>
+      </datatype>
+      <remarks versionDate="2020-03-24" xml:lang="en">
+        <p><att>delim</att> must contain at least one character.</p>
+      </remarks>
+    </attDef>
+    <attDef ident="match" usage="req">
+      <gloss versionDate="2020-02-03" xml:lang="en">match</gloss>
+      <desc versionDate="2020-02-03" xml:lang="en">supplies an XPath expression which returns a sequence of nodes which are 
+        citable structural components. The expression may be absolute (beginning with <code>/</code>) or relative. 
+        <att>match</att> on a <gi>citeStructure</gi> without a <gi>citeStructure</gi> parent must be an absolute XPath. If it is
+        relative, its context is set by the <att>match</att> of the parent <gi>citeStructure</gi>. </desc>
+      <datatype>
+        <dataRef key="teidata.xpath"/>
+      </datatype>
+    </attDef>
+    <attDef ident="unit" usage="opt">
```

Comment:
In the original FR in ticket #1957 the `@unit` attribute was a required attribute. I prefer it being optional because it's merely a label with no functionality, right?

If it's only for documentary purposes, what about adding `<desc>` to the content model of `<citeStructure>`?

Reply:
@peterstadler do you mean instead of using `@unit`? I'd like to keep it, because it's an enumerated list, but I've no objection to adding `<desc>` as well.